mne/channels/tests/test_channels.py

```diff
@@ -312,7 +312,7 @@ def test_set_montage():
     from mne.io import RawArray
     from mne.utils import object_diff
 
-    ch_names = ['EEG {id:03d}'.format(id=id) for id in range(1, 61)]
+    ch_names = ['EEG {id:03d}'.format(id=id) for id in range(61)]
     raw = RawArray(data=np.empty([len(ch_names), 1]),
                    info=create_info(ch_names=ch_names, sfreq=1))
     original_raw = (read_raw_fif(raw_fname, preload=False)
```

Comment:
when reading `chs` contains `EEG 001` - `EEG 060`, but dig contains from 0 to 60. there's no notion of name, but the first one is the one that gets dropped

Reply:
Is this neuromag? The first digitized point is the reference electrode

Reply:
(or maybe it's the last? Can't remember, but you can check by comparing to `info['chs'][eeg_idx]['loc']`)

Reply:
I think ref was first (after cardinals) in our (neuromag) lab.